setup.py

```diff
@@ -75,5 +75,9 @@
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
+        'Programming Language :: Python :: 3.11',
```

Comment:
Yeah, still looks like no 3.11 support.

We try not to claim support of a version until we have the test, so maybe drop 3.11 for now.